save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/service/DockerService.kt

```diff
@@ -175,10 +177,19 @@ class DockerService(private val configProperties: ConfigProperties) {
         val testSuitesDir = resourcesPath.resolve(standardTestSuiteDir)
         // copy corresponding standard test suites to resourcesRootPath dir
         copyTestSuitesToResourcesPath(testSuitesForDocker, testSuitesDir)
+
+        // move additional files, which were downloaded into the root dit to the execution dir for standard suites
+        execution.additionalFiles?.split(";")?.filter { it.isNotBlank() }?.forEach {
+            val additionalFilePath = Paths.get(resourcesPath.resolve(File(it).name).absolutePath)
+            val destination = Paths.get(testSuitesDir.absolutePath).resolve(File(it).name)
+            log.info("Move additional file $additionalFilePath into $destination")
+            Files.move(additionalFilePath, destination, StandardCopyOption.REPLACE_EXISTING)
```

Comment:
When are we copying those files in `resourcesPath`? Maybe, we can move them in the proper directory at that point already?

Reply:
We download files into `resourcesPath` during `uploadBin`, and at those moment  we doesn't know anything about `standardTestSuiteDir`, so I decide to provide logic of movement here